remoulade/api/main.py

```diff
@@ -109,6 +109,28 @@ def get_scheduled_jobs():
     return {"result": [job.as_dict() for job in scheduled_jobs.values()]}
 
 
+@app.route("/scheduled/jobs", methods=["POST"])
+def add_job():
+    job_dict = ScheduledJobSchema().load(request.json)
+    get_scheduler().add_job(job_dict)
```

Comment:
maybe create a `with_scheduler` decorator to return an error if it doesn't exist or inject it if it does